internal/proxy/util.go

```diff
@@ -68,7 +68,7 @@ const (
 	DefaultArithmeticIndexType = "STL_SORT"
 
 	// DefaultStringIndexType name of default index type for varChar/string field
-	DefaultStringIndexType = "Trie"
```

Comment:
Do not edit this for better compatability.

Reply:
This was decided by @czs007 , so all our index types are in uppercase.